src/styles/main.css

```diff
@@ -0,0 +1,69 @@
+.page__body {
+  margin: 0;
+  height: 100vh;
+  background-color: #fff;
```

Comment:
Code Style issue: It's a good practice to use variables for colors. This makes the code more maintainable and readable. Consider defining a CSS variable for this color.